src/index.html

```diff
@@ -3,11 +3,287 @@
+        <article class="section__article">
+
+          <div class="pictures">
+            <img
+              class="pictures__item pictures__item--big"
+              src="images/all-products1.jpg"
+              alt="birds-on-phone"
+            >
+            <img
+              class="pictures__item pictures__item--small"
+              src="images/all-products2.jpg"
+              alt="headphones-box-1"
+            >
+            <h3 class="section__title section__title--second">
+              All products
+            </h3>
+          </div>
+
+
+        </article>
+
+        <article class="section__article">
+
+          <div class="pictures">
+            <img
+              class="pictures__item pictures__item--small"
+              src="images/audio1.jpg"
+              alt="headphones"
+            >
+            <img
+              class="pictures__item pictures__item--big"
+              src="images/audio2.jpg"
+              alt="headphones-box-1"
+            >
+
+            <h3 class="section__title section__title--second">
+              Audio
+            </h3>
+          </div>
+
+
+        </article>
+
+        <article class="section__article">
+
+          <div class="pictures">
+            <img
+              class="pictures__item pictures__item--big"
+              src="images/accessories1.jpg"
+              alt="mobilephone"
+            >
+            <img
+              class="pictures__item pictures__item--small"
+              src="images/accessories2.jpg"
+              alt="phone-charger"
+            >
+
+            <h3 class="section__title section__title--second section__title--button">
+              Accessories
+            </h3>
+          </div>
+
+          <a href="#" class="button">View all</a>
+        </article>
+      </section>
+    </main>
+
+    <div class="separator"></div>
+
+    <footer class="footer">
+      <div class="about">
+        <section class="section">
+          <h2 class="title title__secondiary title__about">
+            About us
+          </h2>
+
+          <p class="section__paragraph">
+            We&#x2019;re a London-based tech company on a mission to remove barriers between people and technology.
+          </p>
+          <p class="section__paragraph">
+            To do this, We&#x2019;re crafting intuitive, flawlessly connected products that improve our lives without getting in the way.
+            No confusing tech-speak. No silly product names. Just artistry, passion and trust. And products We&#x2019;re proud to share with our friends and family. Simple.
+          </p>
+          <p class="section__paragraph">
+            Because beautiful tech shouldn&#x2019;t be complicated.
+          </p>
+        </section>
+      </div>
+
+      <div class="contact">
+
+        <h2 class="title title__secondiary title__contact">
+        Contact us
+        </h2>
+
+        <form class="form contact__form">
+          <input
```

Comment:
Consider adding `name` values for all inputs to make sure the submitted values are always in payload after sending form